_pages/open-source-day-2021-summer-contribution.md

```diff
@@ -13,13 +13,27 @@ As part of the Contribution Track, you will contribute code to curated open sour
 
 ### Partner Projects
 
-Coming soon
+| Project | Description | Maintainer(s) |
+| ------- | ----------- | ------------- |
+| [Apache Sharding Shpere](https://github.com/apache/shardingsphere) | Apache ShardingSphere is an open-source ecosystem consisted of a set of distributed database solutions, including 3 independent products, JDBC, Proxy & Sidecar (Planning). They all provide functions of data scale out, distributed transaction and distributed governance, applicable in a variety of situations such as Java isomorphism, heterogeneous language and cloud native. | [Juan Pan (Trista)](https://github.com/tristaZero) |
+| [Backdrop](https://github.1/backdrop/backdrop) | Backdrop is a full-featured content management system that allows non-technical users to manage a wide variety of content. It can be used to create all kinds of websites including blogs, image galleries, social networks, intranets, and more. | [Jen Lampton](https://github.com/jenlampton) |
+| [Greenpeace Planet 4](https://github.com/greenpeace) | Built on top of Wordpress tech, Greenpeace Planet 4 powers digital platforms to engage with millions and win campaigns around the world. | [Suzi Grishpul](https://github.com/suzi-greenpeace) |
+| [Material-UI](https://github.com/mui-org/material-ui) | Material-UI is a simple and customizable component library to build faster, beautiful, and more accessible React applications. Follow your own design system, or start with Material Design. | [Marija Najdova](https://github.com/mnajdova) |
+| [Openstack](https://opendev.org/openstack) | OpenStack is a cloud operating system that controls large pools of compute, storage, and networking resources throughout a datacenter, all managed through a dashboard that gives administrators control while empowering their users to provision resources through a web interface. | [Amy Marrich]() |
+| [OpenStreetMap](https://github.com/Intuit/Superglue) | TBA | TBA |
+| [OpenSUSE](https://github.com/openSUSE) | TBA | TBA |
```

Comment:
Both openstreetmap and opensuse asked the committees to reach out before posting their info on the website. Leaving it blank for now